Orange/preprocess/randomize.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+
+from Orange.data import Table
+from .preprocess import Randomize
+
+__all__ = ["Randomizer"]
+
+
+class Randomizer:
```

Comment:
This class could be merged into Orange.preprocess.Randomize
making this module obsolete.